schema/definitions/0.8.0/schema/fmu_results.json

```diff
@@ -168,9 +168,7 @@
                 "name",
                 "format",
                 "spec",
-                "stratigraphic",
-                "is_prediction",
-                "is_observation"
+                "stratigraphic"
```

Comment:
In case of surfaces, where properties are not applied (implicit one property), will this be correct?

Reply:
If we decide to move everything into `properties` at some point, this must be changed. But for now it works. It is same as before, except the `is_prediction` has been removed from required entries. Which is fine, I think.